modules/costs/app/views/cost_objects/items/_material_budget_item.html.erb

```diff
@@ -99,5 +99,5 @@ See docs/COPYRIGHT.rdoc for more details.
       <span class="hidden-for-sighted"><%= t(:button_delete) %></span>
     </a>
   </td>
-</tr>
+< tr>
```

Comment:
This change looks like mistake.

Reply:
Good catch, thanks @akasparas :bowing_man: